src/components/Footer/Footer.module.css

```diff
@@ -1,7 +1,58 @@
-p {
-  padding-bottom: 15px;
-}
-
 .button button {
   margin: 10px;
 }
+
+.title__wrap {
+  display: flex;
+  justify-content: space-between;
+}
+
+.title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 100%;
+  margin: 0;
+}
+
+.title__button {
+  display: flex;
+  align-items: center;
+}
+
+.button {
+  display: block;
+  height: 19px;
+  border: none;
+  background: transparent;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 14px;
+  outline: none;
+  cursor: pointer;
+}
+
+.button span {
+  color: #356eff;
+}
+
+.selected span {
+  color: #fff;
+}
+
+.button:hover {
+  background-color: #356eff;
+  color: #fff;
+  border-radius: 37px;
+}
+
+.selected {
+  background-color: #356eff;
+  color: #fff;
+  border-radius: 37px;  
+}
+
+.button:hover span {
+  color: #fff;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5830965)